http/snoop-wrap/build.gradle.kts

```diff
@@ -1,17 +1,74 @@
+
 plugins {
     id("kafkasnoop.kotlin-http-service-conventions")
     id("com.google.cloud.tools.jib")
 }
 
+repositories {
+    mavenLocal()
+    mavenCentral()
+
+    val artifactoryContextUrl = property("artifactoryContextUrl")
+    maven {
+        url = uri("$artifactoryContextUrl/corda-dependencies")
+    }
+
+    maven {
+        url = uri("$artifactoryContextUrl/corda-os-maven")
+        authentication {
+            create<BasicAuthentication>("basic")
+        }
+        credentials {
+            username = System.getenv("CORDA_ARTIFACTORY_USERNAME")
+            password = System.getenv("CORDA_ARTIFACTORY_PASSWORD")
+        }
+    }
+}
+
+val cordaApiDependencies: Configuration by configurations.creating
+cordaApiDependencies.isCanBeConsumed = false
+configurations.add(cordaApiDependencies)
+
+configurations {
+    compileClasspath.get().extendsFrom(cordaApiDependencies)
+}
+
 dependencies {
     implementation(project(":http:serialisation:avro"))
     implementation(project(":http:snoop"))
     implementation(project(":lib:avro"))
     implementation(project(":lib:http"))
+
+
+    val cordaApiVersion = property("cordaApiVersion")
+    cordaApiDependencies(platform("net.corda:corda-api:$cordaApiVersion"))
+    cordaApiDependencies("net.corda:corda-avro-schema:$cordaApiVersion")
 }
 
-// version of KafkaSnoop that wraps the snooper and the deserialiser into one HTTP server.
-jib.to.image = "driessamyn/kafkasnoop-wrap"
+jib {
+    container {
+        mainClass = "kafkasnoop.wrap.AppKt"
+
+        ports = listOf("9000", "9000")
+    }
+    extraDirectories {
+        paths {
+            path {
+                setFrom(file("$rootDir/schemas"))
+                into = "/schemas"
+            }
+        }
+    }
+    to {
+        image = "qa-docker-dev.software.r3.com/kafkasnoop-wrap"
+        tags = setOf(project.version.toString())
```

Comment:
maybe we could add two tags, the version (`0.0.5`) and `latest`. Therefore `latest` will update automatically and always point to the latest release.
Not 100% sure whether this is a best practice or not.

Reply:
I think this will work for as a start. A downside could be that we can potentially break two versions at a time - latest and the last incremented one. But we can always go back a previous version.
I'll make the change